test/modules/location.spec.ts

```diff
@@ -390,6 +396,33 @@ describe('location', () => {
           }
         );
       });
+
+      describe('timeZone', () => {
+        it('should return a random timezone', () => {
+          const actual = faker.location.timeZone();
+          expect(faker.definitions.location.time_zone).toContain(actual);
+        });
+      });
     }
   );
 });
+
+describe('definitions', () => {
+  describe('timeZone', () => {
+    it.each(Object.entries(allLocales))(
+      'locale data for %s should be a subset of the base locale',
+      (locale, data) => {
+        if (locale === 'base') {
+          expect(data.location?.time_zone).toEqual(
+            allLocales.base.date?.time_zone
+          );
+        } else if (data.location?.time_zone != null) {
+          // expected and actual are flipped here
+          expect(allLocales.base.date?.time_zone).toEqual(
+            expect.arrayContaining(data.location?.time_zone)
```

Comment:
this should not be nullish-able

```suggestion
            expect.arrayContaining(data.location.time_zone)
```